Library/.rubocop.yml

```diff
@@ -63,9 +63,9 @@ Homebrew/MoveToExtendOS:
 # `system` is a special case and aligns on second argument, so allow this for formulae.
 Layout/ArgumentAlignment:
   Exclude:
-    - "Taps/*/*/*.rb"
-    - "/**/Formula/*.rb"
-    - "**/Formula/*.rb"
+    - "Taps/**/*.rb"
```

Comment:
I think this can stay as `Taps/*/*/*.rb` everywhere, right? 

This would only apply to top-level Ruby files in a tap. And sharding will only be done within `Casks` and `Formula` directories.

Reply:
Agreed. Applying this will fix the style failure as well, I think.

Reply:
This is more permissive so we should use it for all the excludes. For the include, we can omit this entirely.

Reply:
Nope, with or without this change: we get the same `tap_syntax` failure. Would love some help here!

Reply:
Ignore me: I was looking at an outdated commit. It's still running 🤞🏻

Reply:
It worked, thanks all.